server/routerlicious/README.md

```diff
@@ -1,4 +1,4 @@
-# root
+# Routerlicious
```

Comment:
If this breaks repo policy check I'll go fix that in another PR... This shouldn't have to be called `root`.